postman-generator/src/main/scala/generator/PostmanCollectionGenerator.scala

```diff
@@ -228,6 +217,66 @@ object PostmanCollectionGenerator extends CodeGenerator {
     }
   }
 
+  /**
+    * Prepares dependency entities cleanup phase with DELETE operations.
+    * Fills all path parameters with postman variable references.
+    * If parameter is not specified, it tries to build it from raw path
+    * for example /sth/:id/sth => new 'id' param is created
+    *
+    * @param resolvedService Service with all imports at hand.
+    * @param serviceSpecificHeaders Service specific headers.
+    * @param examplesProvider Examples generator.
+    * @return Dependent entities cleanup folder for postman.
+    */
+  def prepareDependentEntitiesCleanup(
```

Comment:
cannot this operation be merged with `prepareDependentEntitiesSetup` and return a Tuple ?  
I think you are running a lot of the same code `DependantOperationResolver.resolve(resolvedService)`. 
Anyway we can leave how it is right now and refactor later if this possible. Your call ;)

Reply:
agreed. I've combined them into one method that returns a tuple